tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj

```diff
@@ -275,6 +275,14 @@
     <PackageReference Include="AWSSDK.BedrockRuntime" Version="3.7.301.35" Condition="'$(TargetFramework)' == 'net6.0'" />
   </ItemGroup>
 
+  <!-- AWS SDK -->
+  <ItemGroup>
+    <PackageReference Include="AWSSDK.Lambda" Version="3.7.404.1" Condition="'$(TargetFramework)' == 'net462'" />
```

Comment:
Should we be testing an older version of the SDK here (this is the same version as the latest version?

Reply:
Ah, I forgot to do that. Good idea.